cmf-cli/Handlers/PackageType/HelpNgCliPackageTypeHandler.cs

```diff
@@ -103,12 +103,12 @@ public HelpNgCliPackageTypeHandler(CmfPackage cmfPackage) : base(cmfPackage)
                     {
                         DisplayName = $"ng build {package.Key}",
                         Command = "build",
-                        Args = String.Equals(package.Key, this.CmfPackage.PackageId, StringComparison.InvariantCultureIgnoreCase) ? new []{ "--", "--base-href", "$(APPLICATION_BASE_HREF)" } : Array.Empty<string>(),
+                        Args = String.Equals(package.Value.Name, this.CmfPackage.PackageId, StringComparison.InvariantCultureIgnoreCase) ? new []{ "--base-href", "$(APPLICATION_BASE_HREF)" } : Array.Empty<string>(),
```

Comment:
pretty sure this is a bug that was fixed in HTML only, check [here]( https://github.com/criticalmanufacturing/cli/blob/df7956a26293e6698c17265aaa90102c8805f81f/cmf-cli/Handlers/PackageType/HtmlNgCliPackageTypeHandler.cs#L121C29-L121C29). I'd try to use the same approach to keep the code easier to reason with. I have some time set aside soon to test this, but if you have feedback sooner, please share :)